src/panel/category/CategoryPanel.jsx

```diff
@@ -78,11 +73,10 @@ export default class CategoryPanel extends React.Component {
     unListen(this.mapMoveHandler);
   }
 
-  fitMapAndFetch() {
-    const rawBbox = (this.props.bbox || '').split(',');
-    const bbox = rawBbox.length === 4 && [[rawBbox[0], rawBbox[1]], [rawBbox[2], rawBbox[3]]];
-    if (bbox) {
-      window.map.mb.fitBounds(bbox, { animate: false });
+  fitMap() {
+    if (this.props.bbox) {
+      fire('fit_map', parseBboxString(this.props.bbox), true);
```

Comment:
As a consequence of `forceAnimate: true` any Intention with bbox with trigger an animated "flyTo" to fit the map. 
I find this acceptable, but want to be sure that we are aware of this.

Reply:
Good catch, I removed this one and kept the animation on the bbox extension.